lib/order.rb

```diff
@@ -0,0 +1,29 @@
+class Order
+attr_reader :basket, :menu
+
+  def initialize(menu = Menu.new)
+    @basket = []
+    @menu = menu
+  end
+
+
+  def add(dish, quantity)
+    fail "#{dish} is not on the menu!" unless menu.has_dish?(dish)
```

Comment:
Good job accounting for edge cases by adding an error message in case dish is unavailable